resources/assets/js/components/stekpakket/StekPakket.vue

```diff
@@ -0,0 +1,387 @@
+		}
+
+		protected get optieLijst() {
+			const lijst = [];
+			for (const groep of this.opties) {
+				for (const optie in groep.opties) {
+					if (!(groep.opties.hasOwnProperty(optie))) {
+						continue;
+					}
+					if (groep.opties[optie].actief) {
+						lijst.push(optie);
+					}
+				}
+			}
+			return lijst;
+		}
+
+		protected slaOp() {
+			this.laden = true;
+			axios.post(this.opslaan, {
+				basispakket: this.gekozenBasispakket,
+				opties: this.optieLijst,
+			}).then(() => {
+				this.laden = false;
+				this.gewijzigd = false;
+			}).catch(() => {
+				this.laden = false;
+			});
+		}
+	}
+</script>
+
+<style>
+	.container {
+		max-width: 1140px !important;
+	}
+</style>
+
+<style scoped lang="scss">
+	.stekpakket {
+		font-family: "Source Sans Pro", Helvetica, Arial, sans-serif;
+	}
+
+	.beeldmerk {
+		width: 80px;
+	}
+
+	h1 {
+		font-size: 20pt;
+		font-weight: 600;
+		line-height: 24pt;
+		margin: 0;
+	}
+
+	.subtitel {
+		font-size: 14pt;
+		font-weight: 300;
+	}
+
+	p {
+		font-size: 13pt;
+		font-weight: 300;
+	}
+
+	h2 {
+		text-align: center;
+		font-size: 17pt;
+		font-weight: 600;
+		margin-bottom: 15px;
+		margin-top: 30px;
+	}
+
+	.equal {
+		display: flex;
+		display: -webkit-flex;
+		flex-wrap: wrap;
+	}
+
+	.pakket {
+		border: 1px solid #3498db;
+		border-radius: 5px;
+		text-align: center;
+		margin-bottom: 30px;
+		min-height: calc(100% - 30px);
+		background: white;
+		color: black;
+		cursor: pointer;
+		padding: 0 15px 65px;
+		position: relative;
+
+		&:hover, &:focus, &.actief {
+			background: #3498db;
+			color: white;
+			border-top-width: 0;
+
+			.titel {
+				background: white;
+				color: #3498db;
+				padding-top: 3px;
+			}
+
+			.usp i {
+				color: white;
+			}
+		}
+
+		.titel {
+			margin-bottom: 15px;
+			background: #3498db;
+			border-radius: 0 0 3px 3px;
+			color: white;
+			font-size: 12pt;
+			font-weight: 600;
+			display: inline-block;
+			padding: 2px 16px;
+		}
+
+		.usp {
+			font-size: 10pt;
+			font-weight: 300;
+			text-align: left;
+			line-height: 17pt;
+
+			i {
+				color: #2ECC71;
+				margin-right: 8px;
+			}
+		}
+
+		.prijs {
+			position: absolute;
+			width: 100%;
+			left: 0;
+			bottom: 10px;
+			right: 0;
+
+			.getal {
+				font-size: 23pt;
+				font-weight: 600;
+				vertical-align: top;
+				line-height: 24pt;
+
+				span {
+					font-weight: 400;
+					font-size: 13pt;
+					vertical-align: top;
+					line-height: 17pt;
+				}
+			}
+
+			.per {
+				font-weight: 300;
+				font-size: 11px;
+				margin-top: -5px;
+			}
+		}
+	}
+
+	.optellingTitel {
+		font-size: 20pt;
+		font-weight: 600;
+	}
+
+	.optelling {
+		text-align: right;
+
+		.getal {
+			font-size: 23pt;
+			font-weight: 600;
+			vertical-align: top;
+			line-height: 24pt;
+
+			span {
+				font-weight: 400;
+				font-size: 13pt;
+				vertical-align: top;
+				line-height: 17pt;
+			}
+		}
+
+		.per {
+			font-weight: 300;
+			font-size: 11px;
+			margin-top: -5px;
+		}
+	}
+
+	.bevestigen {
```

Comment:
Onder deze knop mist nog een margin. Dit is vooral een ding als een lid een effect aan heeft staan waardoor de achtergrond niet wit is.

Reply:
Opgelost met padding op de column